### ISSUE TITLE
too much output

### ISSUE THREAD

**Issue by stsp** [openSUSE/hwinfo#31]
I use "hwinfo --vbe", but the output is very
large, consisting of many many things, not
only vbe. Of course I can use grep, but I
supposed if --<HARDWARE_ITEM> is specified,
only info about this item is printed.

**Comment by wfeldt** [comment]
Yes, it also includes the (sm)bios listing in the output. It's because some vbe info is represented there.

In hindsight maybe not the best decision but hard to change now without breaking some expectations.
